packages/api/test/cli/codegen/languages/typescript/smoketest.test.ts

```diff
@@ -42,25 +63,25 @@ describe('typescript smoketest', function () {
     this.currentTest.slow(5000);
   });
 
-  if (datasetRegex) {
-    // eslint-disable-next-line mocha/no-setup-in-describe
-    dataset.forEach(({ name, url }) => {
-      if (!new RegExp(datasetRegex).test(name)) {
-        return;
-      }
-
-      it(`should generate an SDK for \`${name}\``, smoketestAPI(url, name));
-    });
+  dataset.forEach(({ name, url }) => {
+    it.skip(`should generate an SDK for \`${name}\``, async function () {
+      const spec = await new OASNormalize(url).validate(true).catch(err => {
+        // If this fails for any reason we should know. Catching and re-throwing here for the sake
+        // of test verbosity.
+        throw err;
+      });
 
-    return;
-  }
+      const oas = Oas.init(spec);
+      await oas.dereference({ preserveRefAsJSONSchemaTitle: true });
 
-  // eslint-disable-next-line no-plusplus
-  for (let idx = START_AT_INDEX; idx < START_AT_INDEX + MAX_APIS_TO_TEST; idx++) {
-    if (!(idx in dataset)) {
-      return;
-    }
+      const ts = new TSGenerator(oas, './path/to/spec.json', name);
```

Comment:
Is this path correct? If the path isn't needed in testing can you just use `undefined` or at least `./path/not/needed`?

Reply:
Changed it to `./path/not/needed.json`. This path is what gets set to the `definition` import in the codegen'd SDKs, and since this smoektest doesn't actualy running an SDK, just checks if codegen worked or not, this isn't needed.